contracts/btc-finality/src/state/config.rs

```diff
@@ -0,0 +1,36 @@
+use derivative::Derivative;
+
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Addr;
+
+use cw_controllers::Admin;
+use cw_storage_plus::Item;
+
+pub(crate) const CONFIG: Item<Config> = Item::new("config");
+pub(crate) const PARAMS: Item<Params> = Item::new("params");
+/// Storage for admin
+pub(crate) const ADMIN: Admin = Admin::new("admin");
+
+/// Config are Babylon-selectable BTC finality configuration
+// TODO: Add / enable config entries as needed
+#[cw_serde]
+pub struct Config {
+    pub denom: String,
+    pub babylon: Addr,
```

Comment:
Does the finality contract needs to know babylon contract address? Probably not needed as Babylon's finality module does not need to talk to BTC light clients?

Reply:
Yes, it does. Slashing is being sent from finality to babylon, for forwarding over IBC. In fact from there the slashing info is also propagated to the staking contract, for accounting (setting FP power to zero).

There's some coupling, but it's acceptable, and normal given the functionality.

Also, at the level of the instantiation logic, there's some back and forth to inform the contracts of the addresses of the other contracts. We can improve on that, by example by instantiating the contracts from the SDK. Will create an issue in `babylon-sdk` to do that in a follow-up.